components/ArticleCard/ArticleCardContent.tsx

```diff
@@ -0,0 +1,58 @@
+import Image from 'next/image';
+
+import { CardContent } from '@/components/ui/card';
+import KebabDropDown from '@/components/KebabDropDown';
+import useUserStore from '@/stores/useUser.store';
+
+/**
+ * @param {boolean} props.isBestCard - 게시글 데이터
+ * @param {string} props.title - 게시글 제목
+ * @param {string | null} props.image - 게시글 이미지
+ * @returns {JSX.Element} 게시글 카드의 제목, 이미지가 포함된 Card Content 컴포넌트
+ */
+function ArticleCardContent({
+  isBestCard,
+  title,
+  image,
+  writer,
+}: {
+  title: string;
+  image: string | null;
+  isBestCard: boolean;
+  writer: {
+    id: number;
+  };
+}) {
+  const { user: userData } = useUserStore();
+  return (
+    <CardContent
+      className={`${isBestCard ? 'max-h-pr-72' : 'h-pr-72 mo:max-h-pr-64'} flex justify-between p-0`}
+    >
+      <p className="article-title line-clamp-2 max-h-pr-56 mo:mb-pr-12 mo:max-h-pr-48">
+        {title}
+      </p>
+      <div className="ml-auto flex">
+        {image !== null && (
+          <div className="relative ml-pr-16 size-pr-72 shrink-0 overflow-hidden rounded-lg border mo:size-pr-64">
+            <Image
+              src={image}
+              alt="게시글 이미지"
```

Comment:
```suggestion

  const hasImage = {
    src: image || '/images/img-null.png',
    border: image && 'border-t-secondary',
  };

  const hasImageStyled = classNames(
    'relative ml-pr-16 size-pr-72 shrink-0 overflow-hidden rounded-lg mo:size-pr-64',
    hasImage.border,
  );

  const isBestCardStyled = classNames(
    'flex justify-between p-0',
    isBestCard ? 'max-h-pr-72' : 'h-pr-72 mo:max-h-pr-64',
  );

  return (
    <CardContent className={isBestCardStyled}>
      <p className="article_title line-clamp-2 max-h-pr-56 mo:mb-pr-12 mo:max-h-pr-48">
        {title}
      </p>
      <div className="ml-auto flex">
        {!isBestCard && (
          <div className={hasImageStyled}>
            <Image
              src={hasImage.src}
              alt="게시글 이미지"
```

이거는 확인하시고 수정하시라고 커밋 푸시 안하고 리뷰로 남깁니다!